.yarnrc.yml

```diff
@@ -1 +1,3 @@
+nodeLinker: node-modules
```

Comment:
Should the node linker be changed? Currently the website seems to be using yarn PnP